DEVELOPERS.md

```diff
@@ -470,30 +440,41 @@ together as a [service](./agent/service.py).
 
 ### The RAP Controller
 
-The RAP Controller has two main entrypoints:
+The RAP Controller has one entrypoint:
 
 - [controller.main](./controller/main.py) polls its database for
     active jobs and takes appropriate action. This involves creating RUNJOB tasks for
     new jobs, creating CANCELJOB tasks for jobs which have been cancelled, retrieving
     associated tasks for running jobs and updating their status.
-- [controller.sync](./controller/sync.py) handles all communication between the job-server and the RAP Controller. It polls the job-server for active JobRequests, updates its local Jobs table accordingly, and then posts back the details of all Jobs associated with the active JobRequests it received.
-The bulk of the work here is done by the
-[create_or_update_jobs](./controller/create_or_update_jobs.py) module.
+
+Jobs are created via the [RAP API](#rap-api) `create` endpoint and the [create_or_update_jobs](./controller/create_or_update_jobs.py) module does the bulk of the work.
 
 Only the Controller has access to its database of Jobs and Tasks.
 
-### The RAP Controller API
+### The RAP Controller APIs
 
 This is a very simple [Django application](./controller.webapp/) that allows external
-applications and users (currently just the Agent) to communicate with the Controller.
+applications and users to communicate with the Controller.
```

Comment:
Maybe add something like:
Users of the RAP Controller APIs may be Agents inside backends (which consume the [Controller Task API](#controller-task-api)) or clients in the outside world e.g. job server (which consume the [RAP API](#rap-api)).